nautobot/extras/api/views.py

```diff
@@ -451,6 +450,67 @@ class JobModelViewSet(
     serializer_class = serializers.JobSerializer
     filterset_class = filters.JobFilterSet
 
+    # I don't see an easy way for swagger_auto_schema to provide a different schema depending on the API version.
+    # Since for now we default to the pre-1.3 API behavior, document that version in the schema for now.
```

Comment:
Yeah this is definitely tricky. Possibly will have to be accounted for at the serializer itself?

Reply:
drf-spectacular allows for providing different schemas for the same API endpoint under different API versions, as far as I can tell at this point without actually implementing it.